CedarJava/src/main/java/com/cedarpolicy/AuthorizationEngine.java

```diff
@@ -81,6 +81,6 @@ public interface AuthorizationEngine {
      * @return The Cedar language major version supported
      */
     static String getCedarLangVersion() {
-        return "3.0";
+        return "3.1";
```

Comment:
probably 3.2 or 4.0?  This depends on FFI changes not available in 3.1 right?

Reply:
I'll update to 4.0 since I'm expecting these changes to be released then.